FirebaseAuth/Tests/Sample/SwiftApiTests/TestsBase.swift

```diff
@@ -36,6 +37,7 @@ class TestsBase: XCTestCase {
   #endif
 
   func signInAnonymously() {
+    _ = MyPrivateClass()
```

Comment:
intended here?

Reply:
Nope, unintentionally checked the `FirebaseAuth/Tests/Sample/` directory in. Reverted.